core/consensus/aura/src/lib.rs

```diff
@@ -70,6 +74,10 @@ type Signature<P> = <P as Pair>::Signature;
 /// handle to a gossip service or similar.
 ///
 /// Intended to be a lightweight handle such as an `Arc`.
+#[deprecated(
+	since = "1.1",
```

Comment:
does this version refer to the crate or the substrate release? I'd argue it belongs to the crate in which case this should be `since = "1.0.1"`, no?

```suggestion
	since = "1.0.1",
```